sdk/core/azure-core/vcpkg/vcpkg.json

```diff
@@ -40,8 +40,7 @@
           "default-features": false,
           "features": [
             "ssl"
-          ],
-          "version>=": "7.48"
```

Comment:
Also update this file?

https://github.com/Azure/azure-sdk-for-cpp/blob/b108bf62354ec391407e05b5fd29fa6196f70f0d/sdk/core/azure-core/vcpkg.json#L39

Reply:
@#$@#$. That's the one I need to update, not the one I did.

Reply:
Not a bad idea - it's not relevant (because that vcpkg.json file is invalid (all vcpkg package manifests with `version>=` need to specify a baseline commit) but it's not a bad change.